### ISSUE TITLE
Kotlin playground not accessible with JAWS screen reader + Chrome/Edge

### ISSUE THREAD

**Issue by bpisch** [JetBrains/kotlin-playground#240]
Hello,
I am using a screen reader software called JAWS with the latest Chrome and Edge browsers. However, when I enter the editor's edit field, the arrow keys (up, down, left, right) don't allow me to read the content of the edit field. The only way I can see what is inside the field is by moving the focus one line below the edit field, but in that case the whole content of the edit field gets announced like a single line. And this way I can't modify the editor's content either because the screen reader thinks the editor is empty. Could you look into this please?
Thank you in advance.
